### DISCUSSION TITLE
CalDAV over DAVdroid sync only meeting tasks

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#14465] [How to] [answered]
I has fixed my apache2 configs and established sync #14387. This works but:

Only meeting events synced - Why?
New event on my Android smartphone added as to system as meeting - How to change default type?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
davx5 doesn't work with Google Calendar to synchronize tasks

https://www.davx5.com/faq/tasks/advanced-task-features https://www.reddit.com/r/androidapps/comments/fi6i1b/tasktodo_app_that_syncs_via_davx5_davdroid/

Google just recently added tasks, maybe that's why there is no synchronization yet

To synchronize tasks you can use:  OpenTasks or Tasks https://doc.yetiforce.com/apps/

**Reply by vovpff** [reply]
Ok. Thank you for detailed answer!
